schema/archive.json

```diff
@@ -0,0 +1,210 @@
+{
+  "collections": {
+    "archives": {
+      "name": "archives",
+      "config": {
+        "displayFields": [
+          "id", "target.product", "target.channel", "target.version",
+          "target.platform", "target.locale"
+        ],
+        "schema": {
+          "title": "Archives",
+          "description": "Mozilla software builds.",
+          "type": "object",
+          "additionalProperties": false,
+          "required": [
+            "id",
+            "source",
+            "download",
+            "target"
+          ],
+          "properties": {
+            "id": {
+              "type": "string",
+              "title": "ID",
+              "description": "Build Hub ID"
+            },
+            "schema": {
+              "type": "integer",
+              "title": "Schema version",
+              "description": "Schema version"
+            },
+            "build": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "build_id": {
+                  "type": "string",
+                  "title": "Build ID",
+                  "description": "Build ID"
+                },
+                "build_date": {
+                  "type": "string",
+                  "title": "Build date",
+                  "description": "i.e: 2017-05-11",
+                  "pattern": "^(19|2[0-5])[0-9]{2}-((0[0-9])|(1[0-2]))-(([0-2][0-9])|(3[0-1]))$"
+                },
+                "sunset_date": {
+                  "type": "string",
+                  "title": "Sunset date",
+                  "description": "i.e: 2017-08-24",
+                  "pattern": "^(19|2[0-5])[0-9]{2}-((0[0-9])|(1[0-2]))-(([0-2][0-9])|(3[0-1]))$"
+                },
```

Comment:
So the plan is to build a schema that meet our expectation rather than trying to fit the current schema.

1. We build a proper schema
2. We upgrade the scripts to fit our schema.

The new fields are the one missing from our schema that are part of the product_version table in socorro.